synapse/lib/agenda.py

```diff
@@ -870,7 +870,9 @@ async def _runJob(self, user, appt):
             success = False
             try:
                 opts = {'user': user.iden, 'view': appt.view, 'vars': {'auto': {'iden': appt.iden, 'type': 'cron'}}}
-                async for node in self.core.eval(appt.query, opts=opts):
+                opts = self.core._initStormOpts(opts)
```

Comment:
i don't think this `_initStormOpts` is needed since 'user' is already populated

Reply:
I will double check.

Reply:
We could omit it; but there is a bit of a "if _initStormOpts does other things in the future, we'll miss out". It's an inexpensive call currently.

Reply:
gotcha, makes sense